python/lsst/ip/isr/overscan.py

```diff
@@ -974,3 +873,473 @@ def debugView(self, image, model, amp=None, isTransposed=True):
+        parallelResults = None
+        if self.config.doParallelOverscan:
+            # This does not need any extensions, as we'll only
+            # subtract it from the data region.
+            parallelOverscanBBox = amp.getRawParallelOverscanBBox()
+            imageBBox = amp.getRawDataBBox()
+
+            maskIm = exposure.getMaskedImage()
+            maskIm = maskIm.Factory(maskIm, parallelOverscanBBox)
+
+            # The serial overscan correction has removed some signal
+            # from the parallel overscan region, but that is largely a
+            # constant offset.  The collapseArray method now attempts
+            # to fill fully masked columns with the median of
+            # neighboring values, with a fallback to the median of the
+            # correction in all other columns.  Filling with neighbor
+            # values ensures that large variations in the parallel
+            # overscan do not create new outlier points.  The
+            # MEDIAN_PER_ROW method does this filling as a separate
+            # operation, using the same method.
+            parallelResults = self.correctOverscan(
+                exposure,
+                amp,
+                imageBBox,
+                parallelOverscanBBox,
+                isTransposed=not isTransposed,
+                leadingToSkip=self.config.leadingRowsToSkip,
+                trailingToSkip=self.config.trailingRowsToSkip,
+            )
+            overscanMean = (overscanMean, parallelResults.overscanMean)
+            overscanMedian = (overscanMedian, parallelResults.overscanMedian)
+            overscanSigma = (overscanSigma, parallelResults.overscanSigma)
+            residualMean = (residualMean, parallelResults.overscanMeanResidual)
+            residualMedian = (residualMedian, parallelResults.overscanMedianResidual)
+            residualSigma = (residualSigma, parallelResults.overscanSigmaResidual)
+
+        parallelOverscanFit = parallelResults.overscanOverscanModel if parallelResults else None
+        parallelOverscanImage = parallelResults.overscanImage if parallelResults else None
+
+        return pipeBase.Struct(imageFit=serialResults.ampOverscanModel,
+                               overscanFit=serialResults.overscanOverscanModel,
+                               overscanImage=serialResults.overscanImage,
+
+                               parallelOverscanFit=parallelOverscanFit,
+                               parallelOverscanImage=parallelOverscanImage,
+                               overscanMean=overscanMean,
+                               overscanMedian=overscanMedian,
+                               overscanSigma=overscanSigma,
+                               residualMean=residualMean,
+                               residualMedian=residualMedian,
+                               residualSigma=residualSigma)
+
+
+class SerialOverscanCorrectionTaskConfig(OverscanCorrectionTaskConfigBase):
+    leadingToSkip = pexConfig.Field(
```

Comment:
Can these be moved to the base config?  I see they're both here and in the associated parallel correction config.

Reply:
These are new names (notice it doesn't specify `rows` or `columns` which are subjective), and I don't want to deprecate the old names for no obvious gain.